Chummer/customdata/Bone Lacing Adds to Body/manifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<manifest>
+  <name>Bone Lacing adds to Body</name>
+  <guid>66d452bf-f603-40ff-b792-08790975fcb7</guid>
```

Comment:
Hmm, I don't know. GUIDs work when things are internal within Chummer, but I kind of feel like custom rulesets might need something more. I brought up hashes once as a kind of way to verify that two people are using the exact same contents of a custom ruleset, but I honestly don't know how I would integrate that into the manifest itself. Maybe have it just be a part of Chummer's checks for finding matching rulesets when loading `CharacterOptions`... I don't know. Could be a stupid idea to throw into the trash.